notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -18,6 +18,59 @@ https://en.wikipedia.org/wiki/List_of_science_fiction_films_of_the_2000s
+prison in a spacecraft. When the spaceship is damaged by comet debris and
+makes an emergency crash landing on an empty desert planet, Riddick escapes.
+However, when predatory alien creatures begin attacking the survivors, Riddick
+joins forces with the surviving crew and other passengers to develop a plan to
+escape the planet.
+
+:: Date
+2000-02-18
+
+:: References
+https://en.wikipedia.org/wiki/Pitch_Black_(film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+what if I were stranded on a deserted planet [The story follows the efforts of the surviving crew and passengers of a crash-landed spaceship to get off a desert planet that was inhabited by hoards of man-eating creatures.]
+desert planet [A main novelty of the film is the baked in perpetual daylight by three suns desert planet on which it is set.]
+human vs. nature [The surviving crew and passengers of the Hunter-Gratzner spaceship struggled to survive on the desert planet on which they crash-landed.]
+trust in a potential foe [Carolyn and Johns took a calculated risk by releasing the dangerous criminal Riddick and accepting him into their party. They remained suspicious and mistrustful of him throughout the course of the film, but needed his help to survive on the planet.]
+human vs. monster [Central to the story was the surviving crew and passengers of the Hunter-Gratzner spaceship trying to not get killed by the man-eating creatures that inhabited the desert planet on which they'd crash-landed.]
+monster [The desert planet was inhabited by grotesque, man-eating creatures.]
+human character stereotype [The hero of the story Riddick was a stereotypical hard-boiled, lone wolf, in-it-for-only-himself Hollywood antihero figure.]
+character metamorphosis [The story pointedly concluded with Riddick stated that "he died somewhere on that planet", meaning that he'd left his old life as an infamous criminal and murderer behind there, and was now a reformed man.]
+speculative ability [A main novelty of the story was that Riddick had surgically altered eyes that allowed him to see in the dark.]
+
+:: Minor Themes
+speculative spaceship [The futuristic interstellar vessel Hunter-Gratzner was transporting 40 passengers in cryostasis.]
+cryogenic technology [The Hunter-Gratzner spaceship was transporting 40 cryo-sleeping passengers.]
+habitable planet in a trinary star system [Three suns featured prominently in the desert planet sky.]
+organized religion [The Muslim preacher Imam and his three young acolytes were traveling to a place called New Mecca.]
+role models [The tomboy Jack immediately took a liking to the hardened criminal Riddick and soon thereafter began to emulate him.]
+fear [The marooned Hunter-Gratzner crew and passengers pointedly discussed their fears, or lack thereof, of the vicious creatures that went flying around the desert planet at night, killing anything in their path.]
+gender bending [The young girl Jack was posing as a boy to avoid attention.]
```

Comment:
Was there more to it or do we really want to use this theme when someone just disguised themselves as the opposite sex?

Reply:
I see there's a precedent for using it this way in [The Two Gentlemen of Verona (1593)](https://www.themeontology.org/story.php?name=play%3A%20The%20Two%20Gentlemen%20of%20Verona%20(1593)). So why not?

Reply:
Let's see. Unfortunately, it appears that scholars have not subjected **Pitch Black** to the same level of scrutiny as they have **The Two Gentlemen of Verona** - a travesty, I know.

But I did update the motivate to include that Jack had taken on a male persona in addition to dressing in men's clothing.

Reply:
Motivation updated accordingly.